$NetBSD: patch-ae,v 1.2 2010/06/08 10:15:32 wiz Exp $

Setting FS in some versions of awk makes it null.  The default FS of
"whitespace" works fine, so no need to redefine it.

--- src/mkerrcodes1.awk.orig	2009-07-23 13:52:35.000000000 +0000
+++ src/mkerrcodes1.awk
@@ -55,7 +55,7 @@
 # as is trailing whitespace.
 
 BEGIN {
-  FS="[\t]+";
+  #FS="[\t]+";
   header = 1;
   if (errnoidx == 0)
     errnoidx = 2;
@@ -67,7 +67,7 @@ BEGIN {
 /^#/ { next; }
 
 header {
-  if ($1 ~ /^[0-9]/)
+  if ($0 ~ /^[0-9]/)
     {
       print "#include <errno.h>";
       print "#ifdef _WIN32";
